streaming/java/streaming-api/src/main/java/org/ray/streaming/api/stream/Stream.java

```diff
@@ -1,37 +1,34 @@
 package org.ray.streaming.api.stream;
 
 import java.io.Serializable;
+import org.ray.streaming.api.Language;
 import org.ray.streaming.api.context.StreamingContext;
 import org.ray.streaming.api.partition.Partition;
 import org.ray.streaming.api.partition.impl.RoundRobinPartition;
 import org.ray.streaming.operator.StreamOperator;
-import org.ray.streaming.python.PythonOperator;
 import org.ray.streaming.python.PythonPartition;
-import org.ray.streaming.python.stream.PythonStream;
 
 /**
  * Abstract base class of all stream types.
  *
  * @param <T> Type of the data in the stream.
  */
 public abstract class Stream<T> implements Serializable {
-  protected int id;
-  protected int parallelism = 1;
-  protected StreamOperator operator;
-  protected Stream<T> inputStream;
-  protected StreamingContext streamingContext;
-  protected Partition<T> partition;
+  private int id;
```

Comment:
Shall these fields except `partition` and `parallelism` as final?

Reply:
Yes, just fixed.